src/frontend/src/components/Timesheet/TimesheetWidget.tsx

```diff
@@ -6,6 +6,46 @@ interface TimesheetWidgetProps {
   timesheetData: TimesheetData;
 }
 
+//i think u guys have an hours work data but this works too
+function calculateHoursWorked(startTime: string, endTime: string, breakDuration: string): string {
```

Comment:
I think it's more reliable to get the data from the google sheet. We have some special rules for calculating this, and I don't want there to be any issues with differences between frontend and db. Let's take this out for now and we will work on bringing this later along with a few other cool changes.